### PR TITLE
Add instructions for adding the Art Bot to PBAB servers

### LINKED ISSUE #169: Add PBAB support.

### PR DESCRIPTION
**Pull request by Asupkay**
This adds instructions to the README on how to add Art Bot to a Discord server and configure it to respond to queries.

I'm going to be testing these instructions with NumbersInMotion from Plottables to make sure they are clear enough to be followed so this is just a first pass.

closes #169

### CODE REVIEW

**Review by ryley-o — APPROVED** [review]
LGTM!

**Review by asaltste — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -141,6 +141,47 @@ Factory projects are automatically handled by artbot and may be queried from the
...
+                "neo",
+                "neogen"
+            ]
+        }
+    }
+}
+```
+
+4. Please update the [contract names](#contract-names) table in the README if you added a new contract to `partnerContract.json`.
+5. Add Asupkay as a reviewer on the pull request.
```
Comment (jakerockland) [inline README.md:182]: We shouldn't need to add instructions for there to be a specific reviewer for a given flow, I'd ask that we remove this and create a group role if it is strictly necessary and ensure instructions are clear enough via a combination of public/internal documentation for anyone to do this (though all PRs need to be reviewed, so I don't think this needs to be an explicit step in the process).

File: README.md [hunk 2]
```diff
@@ -141,6 +141,47 @@ Factory projects are automatically handled by artbot and may be queried from the
 
 An example artbot query for a factory project is: `#146 Pieces of Me`
 
+## PBAB instructions
+
+These instructions explain how to configure Art Bot to serve project data in relevant channels.
+
+1. Invite ArtBot to your server by clicking [here](https://discord.com/oauth2/authorize?client_id=794646394420854824&scope=bot&permissions=19520).
```
Comment (jakerockland) [inline README.md:148]: Would add a note that the person trying to do this step must have appropriate permission in the given server (be an admin IIRC).

File: README.md (hunk 1 above)
Comment (Asupkay) [reply README.md:182]: Yeah this should be simple enough for anyone to review as this is just standard modification of the config. I'll remove this step as it is on par with adding a new curated project.

**Pull request merged**